### PR TITLE
Add hosted API callout to the README

### PR DESCRIPTION
**Pull request by awalker4**
Clean up some of the curl examples.

I think the readme on the whole can flow a bit better, but the priority right now is to document how to hit the hosted api.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,37 @@
...
 This repo implements a pre-processing pipeline for the following documents. Currently, the pipeline is capable of recognizing the file type and choosing the relevant partition function to process the file.
 
-* Various plaintext files: `.txt`, `.eml`, `.html`, `.md`, `.json`
+* Plaintext: `.txt`, `.eml`, `.html`, `.md`, `.json`
 * Images: `.jpeg`, `.png`
 * Documents: `.doc`, `.docx`, `.ppt`, `.pptx`, `.pdf`
 
+## :rocket: Unstructured API
+
+Try our hosted API! It's freely available to use with any of the filetypes listed above. This is the easiest way to get started. If you'd like to host your own version of the API, jump down to the Developer Quickstart Guide.
```
Comment (ryannikolaidis) [inline README.md:17]: nit: can this anchor to `Developer Quickstart Guide`. admittedly it's not THAT far below, just a nicety?

File: README.md [hunk 2]
```diff
@@ -25,15 +52,16 @@ This repo implements a pre-processing pipeline for the following documents. Curr
...
-# See the [Unstructured Quick Start](https://github.com/Unstructured-IO/unstructured#eight_pointed_black_star-quick-start) for the many OS dependencies that are required, if the ability to process all file types is desired.
+See the [Unstructured Quick Start](https://github.com/Unstructured-IO/unstructured#eight_pointed_black_star-quick-start) for the many OS dependencies that are required, if the ability to process all file types is desired.
+
 * Run `make install`
 * Start a local jupyter notebook server with `make run-jupyter` <br />
 	**OR** <br />
 	just start the fast-API locally with `make run-web-app`
 
-#### Extracting whatever from some type of document
+#### Testing the API
```
Comment (ryannikolaidis) [inline README.md:62]: This is really something like `Using the API` or the _local_ API, right? My first assumption was that this was going to be about running tests.

**Review by ryannikolaidis — COMMENTED** [review]
Just very small nits, otherwise looks great!

File: README.md (hunk 1 above)
Comment (awalker4) [reply README.md]: I opened a tab to look up how to do md anchors and then forgot to do it 😆

Comment (ryannikolaidis) [reply README.md]: 🤣

**Review by ryannikolaidis — APPROVED** [review]
LGTM!

**Pull request merged**